TelenavDemo/ViewControllers/DriveSession/DriveSessionViewController.swift

```diff
@@ -131,10 +167,21 @@ private extension DriveSessionViewController {
         audioMessageStack.addArrangedSubview(audioMessageTitle)
         audioMessageStack.addArrangedSubview(audioMessage)
 
+        alertMessageStack.addArrangedSubview(alertMessageTitle)
+        alertMessageStack.addArrangedSubview(alertMessage)
+
+        violationMessageStack.addArrangedSubview(violationWarningTitle)
+        violationMessageStack.addArrangedSubview(violationMessage)
+
         mainLabelStack.addArrangedSubview(addressStack)
         mainLabelStack.addArrangedSubview(speedLimitStack)
         mainLabelStack.addArrangedSubview(countryStack)
         mainLabelStack.addArrangedSubview(audioMessageStack)
+        mainLabelStack.addArrangedSubview(alertMessageStack)
+        mainLabelStack.addArrangedSubview(violationMessageStack)
+
+        mainLabelStack.backgroundColor = .white
```

Comment:
Better do: 
`.white.withAlphaComponent(0.9)`
Because alpha is applied to both background and text